first_hour_on_vwb/working_with_data_collections.ipynb

```diff
@@ -0,0 +1,603 @@
+    "                f\"--id={self.input_workspace_id.value.strip()}\",\n",
+    "                f\"--description=\\\"{self.input_description.value}\\\"\",\n",
+    "                f\"--name={self.input_name.value.strip()}\",\n",
+    "            ]\n",
+    "\n",
+    "            print('Running command to create workspace...')\n",
+    "            print('\\n'.join(commandList))\n",
+    "            print('')\n",
+    "            print(\"Your workspace will be ready in less than one minute...\")\n",
+    "            \n",
+    "            result = subprocess.run(commandList, capture_output = True, text = True, timeout=180, check=True)\n",
+    "            print(result.stderr) if not result.stdout else print(result.stdout)\n",
+    "            \n",
+    "            # refreshWorkspaceContextCommand = \"terra workspace list\"\n",
+    "            # result = subprocess.run(refreshWorkspaceContextCommand, shell = True, capture_output = True, text = True, timeout=30, check=True)\n",
+    "\n",
+    "create_ws_widget = CreateWorkspaceWidget()\n",
+    "display(create_ws_widget.vb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Convert new workspace to data collection\n",
+    "<a id=\"convert-to-dc\"></a>\n",
+    "\n",
+    "Now you'll convert your newly created workspace, to which you have added resources, into a data collection which can be shared with others and added to other workspaces. \n",
+    "Run the cell below to create a widget, then populate the widget's input fields and click the button to convert the workspace to a data collection.\n",
+    "\n",
+    "Widget input parameters include:\n",
+    "- `workspace_id`: Must match the workspace ID of the workspace created above.\n",
+    "- `short-description`: Must be a string. This description will be visible in the Add a Data Collection modal and should summarize the purpose and/or contents of your data collection.\n",
+    "\n",
+    "The output should resemble:\n",
+    "```\n",
+    "Workspace properties successfully updated.\n",
+    "ID:                <STUDY_NAME>_<YYMMDD>_dc_ws\n",
+    "Name:               <STUDY_NAME>-Data-<YYMMDD>\n",
+    "Description:       <DESCRIPTION>\n",
+    "Cloud Platform:    GCP\n",
+    "Google project:    <GOOGLE_PROJECT_ID>\n",
+    "Cloud console:     https://console.cloud.google.com/home/dashboard?project=<GOOGLE_PROJECT_ID>\n",
+    "Properties:\n",
+    "  terra-workspace-short-description: <DESCRIPTION>\n",
+    "  terra-type: data-collection\n",
+    "Created:           YYYY-MM-DD\n",
+    "Last updated:      YYYY-MM-DD\n",
+    "# Resources:       <NUMBER_OF_RESOURCES>\n",
+    "Workspace properties successfully updated.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "class ConvertToDataCollectionWidget(object):\n",
```

Comment:
The pulldown of workspace IDs is nice, but I think we ought to try to auto-set it to the just-created data collection workspace (we should be able to collect that ID from running the previous widget). Else, have the pulldown start out with no item selected so that they need to explicitly choose.

Otherwise, I'm worried that someone (mindlessly clicking) will set the data collection property on the random first workspace in the list.

Reply:
See my proposal in previous comment (:

Reply:
I replied there.